Rakefile

```diff
@@ -177,13 +177,14 @@ if USE_RAKE_COMPILER
 
   ENV['RUBY_CC_VERSION'] ||= '1.8.7:1.9.3:2.0.0:2.1.5:2.2.1'
 
+  # To reduce the gem file size strip mingw32 dlls before packaging
   ENV['RUBY_CC_VERSION'].to_s.split(':').each do |ruby_version|
-    task "copy:ffi_c:i386-mingw32:#{ruby_version}" do |t|
```

Comment:
Yes they are. But unfortunately they are triggered for native builds, too. So it breaks, because `x86_64-w64-mingw32-strip` is typically not available on Windows. I have verified that, with this patch, the stripping still works per `rake gem:windows` and that it is not called for `rake compile` on Windows.